lib/src/pages/home.dart

```diff
@@ -19,19 +15,24 @@ class HomePage extends StatelessWidget {
 	/// 
 	/// Use this for app-wide navigation. For example, to navigate to the reminders
 	/// page, pass in 2 for [pageIndex].
-	const HomePage({this.pageIndex = 0});
+	const HomePage({required this.pageIndex});
 
 	@override
 	Widget build(BuildContext context) => ResponsiveBuilder(
-		builder: (_, LayoutInfo layout, __) => ResponsiveScaffold.navBar(
-			navItems: [
-				Dashboard(),
-				ResponsiveSchedule(),
-				ResponsiveReminders(),
-			],
-			initialNavIndex: pageIndex,
-			drawer: const NavigationDrawer(),
-			secondaryDrawer: const NavigationDrawer(),
-		)
+		// This is a workaround for SportsPage's tab-based view that needs a controller.
```

Comment:
```suggestion
		// Workaround for SportsPage's tab-based view that needs a controller.
```
Sorry, was over the line limit 🙄